doc/Advanced.md

```diff
@@ -1161,9 +1168,49 @@ Make sure that you are able to run with the new workflow before actually templat
 
 ### Include a config file
 
-To include a config file (e.g. named `mch_digits`) add this line: `{% include "mch_digits" %}`
+To include a config file (e.g. named `mch_digits`) add this line : 
+```
+{% include "MCH/mch_digits" %}
+```
+The content of the file `mch_digits` is then copied into the config file. Thus make sure that you include all the commas and stuff. 
+
+There is an example in Staging : components/qc/ANY/any/tpc-pulser-calib-qcmn
```

Comment:
If you document it as an example, perhaps it should be enforced with ansible that it is there?

Reply:
You are correct but then we need a simpler example because I don't want to copy all the TPC files (in the subfolder).